lineapy/plugins/taskgen.py

```diff
@@ -22,10 +22,15 @@ def get_task_graph(
     task_breakdown parameter. This will give the main bulk of tasks that should
```

Comment:
Should we add documentation for the arguments?

Reply:
Don't think this is in scope for this PR, which is supposed to be focused on migration of docs.